src/Core/Config/Adapter/IConfigAdapter.php

```diff
@@ -21,10 +21,12 @@ public function load($cat = "config");
 	 * Get a particular system-wide config variable given the category name
 	 * ($family) and a key.
 	 *
+	 * Note: Boolean variables are defined as 0/1 in the database
+	 *
 	 * @param string  $cat The category of the configuration value
 	 * @param string  $key The configuration key to query
 	 *
-	 * @return mixed Stored value or "!<unset>!" if it does not exist
+	 * @return null|mixed Stored value or null if it does not exist
```

Comment:
I'm curious: Isn't `null` included in `mixed`?

Reply:
I thought it would be helpful to explicit mention that the return could be nullable.

Reply:
Fair enough.